src/test/java/com/conveyal/gtfs/validator/SpeedTripValidatorTest.java

```diff
@@ -112,6 +113,12 @@ public void tripTravelingTooSlowWithMissingStopTimesHasError() {
         checkFeedHasError(TRAVEL_TOO_SLOW, "6", 3);
     }
 
+    @Test
+    public void tripHasDuplicateArrivalTimesHasError() {
```

Comment:
Perhaps change the name to `tripHasDuplicateArrivalTimesError()` and remove `public`.